src/types/Todo.tsx

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
```

Comment:
Consider using a more descriptive name for the boolean property. For example, `isCompleted` instead of `completed`. 🤔